src/utils.py

```diff
@@ -158,10 +161,14 @@ def check_file_integrity(filename):
 
     return result, json_data
 
-def capture_screen_base64str():
+def capture_screen_base64str(persist):
     '''takes a screenshot and returns it as a base64 encoded string'''
     screenshot = ImageGrab.grab()
     buffered = BytesIO()
     screenshot.save(buffered, format="PNG")
+    if persist:
+        current_timestamp = int(time.time())
+        filename = SCREENSHOT_PREFIX + str(current_timestamp) + ".png"
+        screenshot.save(os.path.join(SCREENSHOT_FOLDER, filename), format="PNG")
```

Comment:
You need to create the `Screenshots` folder if it doesn't already exist. 
Something like this should work:
```
if not os.path.exists(SCREENSHOT_FOLDER):
    os.makedirs(SCREENSHOT_FOLDER)
```